src/jit/lower.cpp

```diff
@@ -628,20 +620,14 @@ GenTree* Lowering::LowerSwitch(GenTree* node)
 
                 // Now, build the conditional statement for the current case that is
                 // being evaluated:
-                // GT_JTRUE
-                //   |__ GT_COND
-                //          |____GT_EQ
-                //                 |____ (switchIndex) (The temp variable)
-                //                 |____ (ICon)        (The actual case constant)
-                GenTreePtr gtCaseCond =
-                    comp->gtNewOperNode(GT_EQ, TYP_INT, comp->gtNewLclvNode(tempLclNum, tempLclType),
-                                        comp->gtNewIconNode(i, TYP_INT));
-                /* Increment the lvRefCnt and lvRefCntWtd for temp */
+                // GT_CMP switchValue, caseValue
+                // GT_JCC(EQ)
```

Comment:
Yes.  I'd be interested turning operations of 'or', 'xor' and 'and' of power of 2 (or 'not' pow 2) into their variants of bt{x}.  Some of these cases will feed a compare and just boil down to just the bt feeding the branch, but some won't.  (I'm just cribbing off of the native compiler peep file here)  Question: when you say you have the bit test of 64 bit "flags" working, does that mean you're recognizing trees that collapse into a test of a single bit and turning them into bt?  I was assuming that more of the cases than 64 bits would turn into a size/instruction count win.  

Cases where we should see it to start are like this:

 Replace:
```
       and   eax, X               ; X can be register, memory, or immediate.
       cmp   eax, #0              ; Must be zero.
       jeq   L1                   ; eq or ne, must be last and only use of cc, since the 
                                  ; replacement 'test' sets different flags than the 'cmp'.
```
 With:
```
       test  eax, X
       jeq   L1
```
If X is a power of 2 we can change it to 'bt' 

There's a lot of variants here just looking at the native compiler.

What kind of diffs are you getting out of our test code?